articles/quickstarts/search.md

```diff
@@ -10,6 +10,8 @@ uid: microsoft.quantum.quickstarts.search
 
 # Quickstart: Implement Grover's search algorithm in Q#
 
+In this Quickstart, you can learn how to build and run Grover search to speed up the search of unstructured data.  Grover's search is one of the most popular quantum computing algorithms, and this relatively small Q# implementation gives you a sense of some of the advantages of programming quantum solutions with a high-level Q# quantum programming language to express quantum algorithms.  At the end of the guide, you will see the simulation output demonstrates successfully finding a specific string among a list of onordered entries in a fraction of the time it would take to search the whole list on a classical computer.
```

Comment:
```suggestion
In this Quickstart, you can learn how to build and run Grover search to speed up the search of unstructured data.  Grover's search is one of the most popular quantum computing algorithms, and this relatively small Q# implementation gives you a sense of some of the advantages of programming quantum solutions with a high-level Q# quantum programming language to express quantum algorithms.  At the end of the guide, you will see the simulation output demonstrates successfully finding a specific string among a list of unordered entries in a fraction of the time it would take to search the whole list on a classical computer.
```